wait/host_port.go

```diff
@@ -13,13 +13,21 @@ import (
 	"github.com/docker/go-connections/nat"
 )
 
+type exitStatus = int
+
+const (
+	exitEaccess     exitStatus = 126 // container cmd can't be invoked (permission denied)
+	exitCmdNotFound exitStatus = 127 // container cmd not found/does not exist or invalid bind-mount
+)
```

Comment:
suggestion: As we're not using the type here because its not our data, this can be removed.
```suggestion
const (
	exitEaccess             = 126 // container cmd can't be invoked (permission denied)
	exitCmdNotFound = 127 // container cmd not found/does not exist or invalid bind-mount
)
```

Reply:
Done.